error TS6137: Cannot import type declaration files. Consider importing 'hast' instead of '@types/hast'.

The npm package incorrectly [uses](https://app.unpkg.com/hastx@0.0.11/files/esm/jsx-runtime.d.ts#L1)

```ts
import type * as hast from "@types/hast";
```

which causes TypeScript errors with `"module": "nodenext"`. This should be

```ts
import type * as hast from "hast";
```

Related:

- denoland/dnt#457
- denoland/deno#27257

## Minimal reproduction

**`tsconfig.json`**

```json
{
  "compilerOptions": {
    "module": "nodenext",
    "noEmit": true,
    "jsx": "react-jsx",
    "jsxImportSource": "hastx",
  }
}
```

**`hello.tsx`**

```tsx
const hello = <h1>Hello, world!</h1>;
```

```console
$ npm i hastx typescript
$ npx tsc
node_modules/hastx/script/html.d.ts:1:28 - error TS6137: Cannot import type declaration files. Consider importing 'hast' instead of '@types/hast'.

1 import type * as hast from "@types/hast";
                             ~~~~~~~~~~~~~

node_modules/hastx/script/jsx-runtime.d.ts:1:28 - error TS6137: Cannot import type declaration files. Consider importing 'hast' instead of '@types/hast'.

1 import type * as hast from "@types/hast";
                             ~~~~~~~~~~~~~

node_modules/hastx/script/svg.d.ts:1:28 - error TS6137: Cannot import type declaration files. Consider importing 'hast' instead of '@types/hast'.

1 import type * as hast from "@types/hast";
                             ~~~~~~~~~~~~~


Found 3 errors in 3 files.

Errors  Files
     1  node_modules/hastx/script/html.d.ts:1
     1  node_modules/hastx/script/jsx-runtime.d.ts:1
     1  node_modules/hastx/script/svg.d.ts:1
```

--- a/deno.json
+++ b/deno.json
@@ -29,6 +29,7 @@
     ]
   },
   "imports": {
+    "hast": "npm:@types/hast@^3.0.0",
     "hastx/": "./",
     "hastx/jsx-runtime": "./jsx-runtime.ts"
   },

--- a/html.ts
+++ b/html.ts
@@ -1,4 +1,4 @@
-import type * as hast from "npm:@types/hast@^3.0.0";
+import type * as hast from "hast";
 
 export type HTMLChild =
   | string

--- a/jsx-runtime.ts
+++ b/jsx-runtime.ts
@@ -1,4 +1,4 @@
-import type * as hast from "npm:@types/hast@^3.0.0";
+import type * as hast from "hast";
 import type * as html from "./html.ts";
 import type * as svg from "./svg.ts";
 

--- a/svg.ts
+++ b/svg.ts
@@ -1,4 +1,4 @@
-import type * as hast from "npm:@types/hast@^3.0.0";
+import type * as hast from "hast";
 
 export type SVGChild =
   | string

--- a/tasks/build-npm.ts
+++ b/tasks/build-npm.ts
@@ -57,11 +57,8 @@ await build({
       node: ">= 16",
     },
     sideEffects: false,
-  },
-  mappings: {
-    "npm:@types/hast@^3.0.0": {
-      name: "@types/hast",
-      version: "^3.0.0",
+    dependencies: {
+      "@types/hast": "^3.0.0",
     },
   },
 });

